asm/trafficdirector/td-galley.yaml

```diff
@@ -61,7 +61,7 @@ spec:
         - --validation-port=9443
         - --sinkAuthMode=$(SINKAUTHMODE)
         - --sinkAddress=$(SINKADDRESS)
-        - --sinkMeta=project_id=$(PROJECT_ID),sds_path=$(SDSPATH),trace=0,enable_security_beta=false # {"$ref":"#/definitions/io.k8s.cli.substitutions.sink-meta"}
+        - --sinkMeta=project=$(PROJECT_ID),location=$(GCP_LOCATION),network=projects/$(PROJECT_ID)/global/networks/default,sds_path=$(SDSPATH),trace=0,enable_security_beta=false # {"$ref":"#/definitions/io.k8s.cli.substitutions.sink-meta"}
```

Comment:
network=projects/$(NETWORK_PROJECT_ID)/global/networks/$(NETWORK_NAME)

Reply:
These are actually set by KPT directly. The $(PROJECT_ID) here is because it is using the project ID interpolation from the env variables that are passed. If a user set the kpt setter for network project, it would be replaced by the value of that setter as a plain string (not as an interpolated variable).